services/web-app/components/nav-tree/nav-tree.js

```diff
@@ -74,6 +79,7 @@ const NavTree = ({ activeItem, items = [], label }) => {
             key={node.title}
             onClick={() => node.path && router.push(node.path)}
             isExpanded={isTreeNodeExpanded(node)}
+            className={clsx({ [styles.section]: node.addMarginTop })}
```

Comment:
```suggestion
            className={clsx({ [styles.section]: node.isSection })}
```

Reply:
I think it's `node.isSection || node.addMarginTop`

Reply:
@francinelucca ?? I'm confused.